File: schemacrawler-scripting/src/test/resources/plaintextschema.js
var forEach = Array.prototype.forEach;

print(catalog.getCrawlInfo())

forEach.call(catalog.getTables(), function(table)
{
  print('');
  print(table.getFullName());
  forEach.call(table.getColumns(), function(column)
  {
    print("  " + column.getName());
  });
});

File: schemacrawler-scripting/src/test/resources/plaintextschema.rb
# encoding: utf-8
puts catalog.getCrawlInfo().toString()

catalog.getTables().each do |table|
  # $stderr.puts table.getSchema().toString()
  if table.getSchema().toString() == "PUBLIC.FOR_LINT"
    next
  end
  # $stderr.puts table.getFullName()
  puts ''
  puts table.getFullName()
  table.getColumns().each do |column|
    puts "  " + column.getName()
  end
end
